L2-counter/hardhat.config.ts

```diff
@@ -0,0 +1,30 @@
+require('@matterlabs/hardhat-zksync-deploy');
+require('@matterlabs/hardhat-zksync-solc');
+
+module.exports = {
+  zksolc: {
+    version: '1.2.0',
+    compilerSource: 'binary', // use "docker" if you dont have zksolc installed globally
```

Comment:
u -> U

Reply:
fixed in c524d3a1470771290f717b27c219f20ecab10e72